### PR TITLE
Task/34410 - Upload multiple images to address and show address use f…

### PR DESCRIPTION
**Pull request by cuongnguyen2503**
…ancybox

### CODE REVIEW

File: app/views/addresses/show.html.erb [hunk 1]
```diff
@@ -1,3 +1,21 @@
...
+    <div class="title">
+      <%= @address.address %>
+    </div>
+    <div class="address-content">
+      <%= render "addresses/carousel", address: @address %>
+    </div>
+  </div>
+  <div class="address-reviews">
+    <div class="user-info">
+
```
Comment (bapboy18) [inline app/views/addresses/show.html.erb]: space

File: app/assets/javascripts/googlemapapi.js [hunk 2]
```diff
@@ -128,7 +128,7 @@ function attachSecretMessage(marker, num) {
   google.maps.event.addListener(marker, "click", function() {
     $.fancybox({
       "type" : "iframe",
-      "href" : gethost() +"/addresses/"+ addresses[num].id,
+      "href" : "http://" + location.host +"/addresses/"+ addresses[num].id,
```
Comment (daothanhcam) [inline app/assets/javascripts/googlemapapi.js]: Missing spaces.

File: app/views/addresses/_carousel.html.erb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <% end %>   
+  </div>
+
+  <a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
+    <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
+  </a>
+  <a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
+    <span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
+    <span class="sr-only">Next</span>
```
Comment (samnangcattor) [inline app/views/addresses/_carousel.html.erb]: I18n .

File: app/views/addresses/_carousel.html.erb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    <% address.images.each do |image| %>
+      <div class="item">
+        <%= image_tag image.photo_url %>
+      </div>      
+    <% end %>   
+  </div>
+
+  <a class="left carousel-control" href="#carousel-example-generic" role="button" data-slide="prev">
+    <span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
+    <span class="sr-only">Previous</span>
```
Comment (samnangcattor) [inline app/views/addresses/_carousel.html.erb]: I18n

File: app/controllers/addresses_controller.rb [hunk 5]
```diff
@@ -5,32 +5,38 @@ class AddressesController < ApplicationController
...
 
   def show
     @address = Address.find params[:id]
+    @images = @address.images.all
     render layout: "address"
   end
 
   def new
     @address = Address.new
+    @image = @address.images.build
```
Comment (samnangcattor) [inline app/controllers/addresses_controller.rb]: create the method to call @image = @address.image.build

Comment (cuongnguyen2503) [inline app/controllers/addresses_controller.rb]: Write that you mention come wrong.

**Pull request merged**